### PR TITLE
Feature/ui

### PR DESCRIPTION
**Pull request by kyakusahmed**

### CODE REVIEW

**Review by mendozabree — COMMENTED** [review]
@kyakusahmed nice work.
Please make these changes
- In the `inbox` when someone reads an email there shouldn't be an `edit` button
- In `drafts` when the `edit` icon has been clicked a page should be loaded when one can edit
- As an admin once someone clicks `groups` they can not create a new group consider adding a button
- Style the button for `add member`

**Review by mendozabree — COMMENTED** [review]
image
Please position in the `form for an email` so it doesn't overlap, let it fit in the white background

File: UI/ad-grp.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <a href="signin.html">Epic Email</a>   
+    </div>
+    <a href="signin.html" class="nav">SignOut</a>
+    <a href="admin.html" class="nav">Home</a>
+</div>
+<div class="whole">
+    <div class="left">
+        <a class="active" href="#home">Groups<span>(2)</span></a>
+    </div>
+
```
Comment (mendozabree) [inline UI/ad-grp.html]: Remove the blank lines from this file

File: UI/draft.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+</div>
+<div class="whole">
+    <div class="left">
+        <a href="new.html">New Message</a>
+        <a href="main.html">Inbox<span> (4)</span></a>
+        <a href="sent.html">sent<span> (2)</span></a>
+        <a class="active" href="#home">Draft<span>(2)</span></a>
+        <a href="group.html">Groups<span>(2)</span></a>
+    </div>
+
```
Comment (mendozabree) [inline UI/draft.html]: Remove the blank lines in this file

File: UI/group.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+</div>
+<div class="whole">
+    <div class="left">
+        <a href="new.html">New Message</a>
+        <a href="main.html">Inbox<span> (4)</span></a>
+        <a href="sent.html">sent<span> (2)</span></a>
+        <a href="draft.html">Draft<span>(2)</span></a>
+        <a class="active" href="#home">Groups<span>(2)</span></a>
+    </div>
+
```
Comment (mendozabree) [inline UI/group.html]: Remove the blank lines from this file

File: UI/main.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+</div>
+<div class="whole">
+    <div class="left">
+        <a href="new.html">New Message</a>
+        <a class="active" href="#home">Inbox<span> (4)</span></a>
+        <a href="sent.html">sent<span> (2)</span></a>
+        <a href="draft.html">Draft<span>(2)</span></a>
+        <a href="group.html">My groups<span>(2)</span></a>
+    </div>
+
```
Comment (mendozabree) [inline UI/main.html]: Remove the blank lines from this file

File: UI/members.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    </div>
+    <a href="signin.html" class="nav">SignOut</a>
+    <a href="admin.html" class="nav">Home</a>
+    <a class="active" href="#home">Create Group</a>
+</div>
+<div class="whole">
+    <div class="left">
+        <a class="active" href="#home">Groups<span>(2)</span></a>
+    </div>
+
```
Comment (mendozabree) [inline UI/members.html]: Remove the blank lines from this file

File: UI/ad-grp.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (mendozabree) [inline UI/ad-grp.html]: Indent the contents of the `head` tag for all files
Something like this
```
<head>
    <title>Epic Email</title>
    <link rel="stylesheet" type="text/css" href="css/style.css">
    <meta charset="utf-8"> 
    <META name="keywords" lang="en-us" content="emails">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>
```

**Review by mendozabree — CHANGES_REQUESTED** [review]

File: UI/ad-grp.html (hunk 6 above)
Comment (kyakusahmed) [reply UI/ad-grp.html]: Resolved

File: UI/members.html (hunk 5 above)
Comment (kyakusahmed) [reply UI/members.html]: blank lines are removed

File: UI/main.html (hunk 4 above)
Comment (kyakusahmed) [reply UI/main.html]: blank lines are removed

File: UI/group.html (hunk 3 above)
Comment (kyakusahmed) [reply UI/group.html]: blank lines are removed

File: UI/draft.html (hunk 2 above)
Comment (kyakusahmed) [reply UI/draft.html]: Blank lines are removed

File: UI/ad-grp.html (hunk 1 above)
Comment (kyakusahmed) [reply UI/ad-grp.html]: Blank lines are removed

**Review by mendozabree — APPROVED** [review]
LGTM

**Pull request merged**